Library/Homebrew/cmd/pull.rb

```diff
@@ -190,6 +190,10 @@ def pull
           safe_system 'brew', install, '--debug', f.name
         end
       end
+
+      unless ARGV.include? "--no-repair"
```

Comment:
Sure thing. Will chase that down in a little bit. Need to stop procrastinating and do some more essay first :smile_cat: